style.md

```diff
@@ -1042,6 +1043,129 @@ func TestSigner(t *testing.T) {
 </td></tr>
 </tbody></table>
 
+### Avoid Embedding Types in Public Structs
+
+These leak implementation details, inhibit type evolution, and obscure
+documentation.
+
+Assuming you have implemented a variety of list types using a shared
+`AbstractList`, avoid embedding the `AbstractList` in your concrete list
+implementations.
+Instead, hand-write only the methods to your concrete list that they will
+delegate to the abstract list.
+
+```go
+type AbstractList struct {}
+
+// Add adds an entity to the list.
+func (l *AbstractList) Add(e Entity) {
+    // ...
+}
```

Comment:
nit: [We're using 2-space indentation](https://raw.githubusercontent.com/uber-go/guide/master/style.md) for code examples.